### PR TITLE
fix controller tests

### PR DESCRIPTION
**Pull request by cob16**
by removing getUserDataProperty stub

### CODE REVIEW

File: lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js [hunk 1]
```diff
@@ -163,89 +164,109 @@ test('When all previous required questions have been answered - and a url is ret
...
+test('it attaches a user submission when the property is set to true', async t => {
   getInstancePropertyStub.withArgs('service', 'attachUserSubmission').returns(true)
 
   const pageSummaryController = proxyquire('./page.summary.controller', {
     '../../../../page/check-submits/check-submits': {checkSubmits: checkSubmitsStub},
     '../../../../presenter/pdf-payload': {pdfPayload: pdfPayloadStub},
-    '../../../../middleware/routes-output/submission-data-with-labels': {submissionDataWithLabels: submissionDataWithLabelsStub}
+    '../../../../middleware/routes-output/submission-data-with-labels': {submissionDataWithLabels: submissionDataWithLabelsStub},
+    '../../../../constants/constants': {
+      SERVICE_OUTPUT_EMAIL: 'bob@gov.uk'
```
Comment (emileswarts) [inline lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js]: Is setting this constant required for this assertion?

File: lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js [hunk 2]
```diff
@@ -163,89 +164,109 @@ test('When all previous required questions have been answered - and a url is ret
...
+
   t.deepEquals(submissions[1].attachments, [])
   submitterClientSpy.resetHistory()
   t.end()
 })
 
-test.skip('when there are files it attaches them to json submission', async t => {
-  getUserDataPropertyStub.resetHistory()
-
+test('when there are files it attaches them to json submission', async t => {
```
Comment (emileswarts) [inline lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js]: This description doesn't seem to match the assertion, unless I'm missing something?

File: lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js [hunk 3]
```diff
@@ -304,12 +327,30 @@ test.skip('TEAM PDF: it attaches PDF contents to email submission attachments',
...
+  const submissions = submitterClientSpy.getCall(0).args[0].submissions
+  const emailEntries = submissions.filter(s => s.type === 'email')
+  const pdfData = emailEntries[0].attachments[0].pdf_data
+
+  t.deepEquals(pdfData, {some: 'PDF contents'})
+  submitterClientSpy.resetHistory()
+  t.end()
+})
+
+test('team email body can be overiden to inculed a rendered template', async t => {
```
Comment (emileswarts) [inline lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js]: Some typos here but also not sure what it means to include a rendered template?

"team email body can be overridden" should be sufficient no?

File: lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js (hunk 1 above)
Comment (tommotaylor) [reply lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js]: Yeah it is strangely

File: lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js (hunk 2 above)
Comment (tommotaylor) [reply lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js]: Theres a few assertions, it matches one of them, which is probably the most important one.

**Pull request merged**